apollo-runtime/src/commonMain/kotlin/com/apollographql/apollo3/network/ws/WebSocketNetworkTransport.kt

```diff
@@ -238,9 +242,12 @@ private constructor(
     }
   }
 
+  @OptIn(ApolloInternal::class)
   override fun <D : Operation.Data> execute(
       request: ApolloRequest<D>,
   ): Flow<ApolloResponse<D>> {
+    val deferredJsonMergers = mutableMapOf<String, DeferredJsonMerger>()
```

Comment:
Could you use a `val deferredJsonMerger: DeferredJsonMerger` here? This coroutine will only ever see this request so it shouldn't need to care about others?

Reply:
You're totally right! It's a lot simpler like this. (Still need to reset it after the last payload, so I added a reset method)